examples/01_plotting/plot_surf_atlas.py

```diff
@@ -134,7 +134,7 @@
 corr = np.zeros((n_parcels, n_parcels))
 n_parcels_hemi = n_parcels // 2
 corr[np.arange(n_parcels_hemi), np.arange(n_parcels_hemi) + n_parcels_hemi] = 1
-corr = corr + corr.T
+corr += corr.T
```

Comment:
I have seen python beginners getting confused by some of the typical python shortcut, so I wonder if we should not avoid such shortcuts in the examples?

I also totally acknowledge that I may be responsible for adding quite a few python 'idioms' in the example recently so I am not 100% consistent here.

Reply:
I take your point.

Reply:
On the other hand '+=' has its sources in C - not really an Python idiom.

Reply:
yeah yeah I know: but to someone new to programming, they are likely to think of it as "a python thing" (especially as such syntax does not exist in R or matlab, the 2 other languages our users are likely to know a bit about).

as I said: I am not 100% consistent on this, so I'd be happy to leave to those changes in for this PR and open an issue to try to make sure we avoid 'complex syntax' (to be defined) in the examples (and maybe revert those changes later in another PR).

I think this is probably we should discuss witht the core dev to decide.

Reply:
Not a problem to remove this specific commit, if that's what you're asking.

Reply:
na that's ok, leave them: we will talk with the core dev about this and probably try to simplify examples later